docs/firstobject.md

```diff
@@ -76,78 +76,84 @@ There is of course a lot more to Nodes, Content Types, Formatters, Widgets and i
 </div>
 </details>
 
-![Add Content](../imgs/Add-content.jpg)
+# Adding Content
 
-Here you see all the `Content Types` defined by default in Archipelago. Yes, you got it right, go on and click on `Digital Object` to get your first Digital Object Node.
+Below you see all the `Content Types` defined by default in Archipelago. Let's click on `Digital Object` to get your first Digital Object Node.
 
-## That first Digital Object
+![Add Content](../imgs/ingest-first-object_beta2/04_add-content.jpg)
 
 ### My Metadata
 
-What you see here is a `Form Mode` in action, with the Title field exposed as a simple text input (`Field Widget`), a Multi Step Web Form that will ingest metadata into a field of type Strawberry Field (where all the magic happens) 
-attached to that field using a `Webform Field Widget`, an editorial/advanced Block on the right side and a Workflow/state drop down (`Save as`) at the bottom. 
+What you see below is a `Form Mode` in action, with the Title field exposed as a simple text input (`Field Widget`), a Multi Step Web Form that will ingest metadata into a field of type Strawberry Field (where all the magic happens) attached to that field using a `Webform Field Widget`, an editorial/advanced Block on the right side and a Workflow/state drop down (`Save as`) at the bottom.
```

Comment:
This is incorrect and does not match your screenshot. Previously on beta1 we had the node title field exposed on top. Now the title field is hidden and the one visible is the webform element, which does not require any explanation.

Reply:
Whoops, yup - missed this in my first pass. thanks!